### PR TITLE
Add element panel

### PR DESCRIPTION
**Pull request by keithtxw**
**Changes**
- Add element panel and populate with the text field category element selector buttons (static)

**Additional information**

-   You may refer to this [ticket](https://jira.ship.gov.sg/browse/CCUBE-1042)

### CODE REVIEW

File: src/components/side-panel/add-elements-panel/add-elements-panel.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Color } from "@lifesg/react-design-system/color";
+import { Divider } from "@lifesg/react-design-system/divider";
```
Comment (weili-govtech) [inline src/components/side-panel/add-elements-panel/add-elements-panel.styles.tsx]: remove unused import

File: src/components/side-panel/add-elements-panel/add-elements-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+} from "./add-elements-panel.styles";
+
+export const AddElementsPanel = () => {
+    // =============================================================================
+    // RENDER FUNCTIONS
+    // =============================================================================
+    const renderElementButtons = (elementTypes: EElementType[]) => {
+        return elementTypes.map((elementType) => {
+            return (
+                <li key={EElementType[elementType]}>
```
Comment (weili-govtech) [inline src/components/side-panel/add-elements-panel/add-elements-panel.tsx]: `EElementType[elementType]` is returning undefined as `elementType` is not referencing the key of the enum

File: src/components/side-panel/side-panel.tsx [hunk 3]
```diff
@@ -1,22 +1,35 @@
...
+    // =========================================================================
+    const getPanelContent = () => {
+        switch (currentMode) {
+            case EBuilderMode.ADD_ELEMENT:
+                return <AddElementsPanel />;
+            default:
+                return <>Some content here...</>;
+        }
+    };
+
```
Comment (weili-govtech) [inline src/components/side-panel/side-panel.tsx]: this should be under render functions

**Review by weili-govtech — COMMENTED** [review]
there are a couple of broken stories

`stories/card/card.mdx`
- Canvas of Element Selector Card is referencing an invalid story

`stories/card/card.stories.tsx`
- elements in BuilderContext need to be updated from Map to object

File: src/components/side-panel/add-elements-panel/add-elements-panel.tsx (hunk 2 above)
Comment (keithtxw) [reply src/components/side-panel/add-elements-panel/add-elements-panel.tsx]: Yea actually i need not do this. Just `elementType` will do. Erroneous

File: stories/card/card.mdx [hunk 4]
```diff
@@ -9,4 +9,4 @@ import * as CardStories from "./card.stories";
 
 # Element Selector Card
 
-<Canvas of={CardStories.ElementSCardStories.elector} />
\ No newline at end of file
+<Canvas of={CardStories.ElementSCardStories} />
```
Comment (weili-govtech) [inline stories/card/card.mdx]:
```suggestion
<Canvas of={CardStories.ElementSelector} />
```

**Review by weili-govtech — CHANGES_REQUESTED** [review]

File: stories/card/card.mdx (hunk 4 above)
Comment (keithtxw) [reply stories/card/card.mdx]: Oops sorry missed this

**Review by weili-govtech — APPROVED** [review]

**Pull request merged**